server/sonar-server/src/main/java/org/sonar/server/ui/PageDecorations.java

```diff
@@ -41,6 +42,6 @@ public PageDecorations() {
   }
 
   public PageDecoration[] get() {
-    return decorations;//NOSONAR expose internal representation
+    return decorations;// NOSONAR expose internal representation
```

Comment:
![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Is //NOSONAR used to exclude false-positive or to hide real quality flaw ?[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ANoSonar)